data/crac-io/crac-io-csa-profiles/src/main/java/com/powsybl/openrao/data/cracio/csaprofiles/parameters/CsaCracCreationParameters.java

```diff
@@ -9,16 +9,25 @@
 import com.powsybl.commons.extensions.AbstractExtension;
 import com.powsybl.openrao.data.cracapi.parameters.CracCreationParameters;
 
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
 
 /**
  * @author Mohamed Ben-rejeb {@literal <mohamed.ben-rejeb at rte-france.com>}
  */
 public class CsaCracCreationParameters extends AbstractExtension<CracCreationParameters> {
+    private static final String REE = "REE";
+    private static final String REN = "REN";
+    private static final String RTE = "RTE";
+    private static final String ES_FR = "ES-FR";
+    private static final String ES_PT = "ES-PT";
```

Comment:
I think some of these constants strings are already in craccreator.constants package. If there aren't, I think you should add it.